src/PyCall.jl

```diff
@@ -708,6 +708,14 @@ for (mime, method) in ((MIME"text/html", "_repr_html_"),
 end
 
 #########################################################################
+# Expose Python docstrings to Julia doc system
+
+if isdefined(Docs, :getdoc)  # Introduced in Julia .6
+    Docs.getdoc(o::PyObject) = Markdown.parse(o[:__doc__])
```

Comment:
One thing that worries me is that docstrings may often contain lists that are supposed to be one item per line, and Markdown parsing will convert that into a single paragraph.

e.g. [PEP 257](https://www.python.org/dev/peps/pep-0257/) recommends an example:
```py
def complex(real=0.0, imag=0.0):
    """Form a complex number.

    Keyword arguments:
    real -- the real part (default 0.0)
    imag -- the imaginary part (default 0.0)
    """
    if imag == 0.0 and real == 0.0:
        return complex_zero
    ...
```
and the Markdown parsing gives:
```
julia> Markdown.parse(s)
  Form a complex number.

  Keyword arguments: real – the real part (default 0.0) imag – the imaginary
  part (default 0.0)
```